build/repo.targets

```diff
@@ -37,6 +37,9 @@
   <PropertyGroup>
     <BenchmarksDependsOn Condition="'$(NoBuild)' != 'true'">Compile</BenchmarksDependsOn>
   </PropertyGroup>
+  
+  <!-- Used by BenchmarksServer to install latest SDK and runtime but do nothing else -->
+  <Target Name="NoOp" />
```

Comment:
`build.cmd` will install the latest SDK and runtime, but it needs **some** target to build.  The default target would build the whole Benchmarks repo which is a waste of time.  The `NoOp` target does nothing so is as fast as possible.